From fc62a50f280cf384d91698307023e9e2739acb3f Mon Sep 17 00:00:00 2001
From: Xiaotian Wu <wuxiaotian@loongson.cn>
Date: Thu, 8 Apr 2021 09:14:11 +0800
Subject: [PATCH 4/4] glibc-2.28: Remove risc-v strings

    sysdeps/loongarch:
        Makefile
        __longjmp.S
        dl-machine.h
        dl-tls.h
        dl-trampoline.S
        gccframe.h
        jmpbuf-offsets.h
        jmpbuf-unwind.h
        ldsodefs.h
        libc-tls.c
        machine-gmon.h
        memusage.h
        setjmp.S
        sotruss-lib.c
        stackinfo.h
        tst-audit.h

    sysdeps/loongarch/bits:
        link.h
        setjmp.h

    sysdeps/loongarch/lp64/rvd:

        s_ceil.c
        s_floor.c

    sysdeps/loongarch/nptl:
        pthread-offsets.h
        pthreaddef.h
        tls.h

    sysdeps/loongarch/nptl/bits:
        pthreadtypes-arch.h
        semaphore.h

    sysdeps/unix/sysv/linux/loongarch:

        atomic-machine.h
        clone.S
        dl-static.c
        init-first.c
        kernel-features.h
        ldsodefs.h
        libc-vdso.h
        makecontext.c
        profil-counter.h
        sigcontextinfo.h
        syscall.c

    sysdeps/unix/sysv/linux/loongarch/lp64:
        jmp_buf-macros.h

    sysdeps/unix/sysv/linux/loongarch/sys:
        sys/procfs.h
        sys/ucontext.h

Change-Id: Id748a58af9377cbf90b3cee0766bde5c9aaa2183
---
 sysdeps/loongarch/Makefile                              | 2 +-
 sysdeps/loongarch/__longjmp.S                           | 4 ++--
 sysdeps/loongarch/bits/link.h                           | 6 +++---
 sysdeps/loongarch/bits/setjmp.h                         | 4 ++--
 sysdeps/loongarch/dl-machine.h                          | 6 +++---
 sysdeps/loongarch/dl-tls.h                              | 4 ++--
 sysdeps/loongarch/dl-trampoline.S                       | 3 +--
 sysdeps/loongarch/gccframe.h                            | 4 ++--
 sysdeps/loongarch/jmpbuf-offsets.h                      | 4 ++--
 sysdeps/loongarch/jmpbuf-unwind.h                       | 4 ++--
 sysdeps/loongarch/ldsodefs.h                            | 2 +-
 sysdeps/loongarch/libc-tls.c                            | 6 +++---
 sysdeps/loongarch/lp64/rvd/s_ceil.c                     | 4 ++--
 sysdeps/loongarch/lp64/rvd/s_floor.c                    | 4 ++--
 sysdeps/loongarch/machine-gmon.h                        | 4 ++--
 sysdeps/loongarch/memusage.h                            | 4 ++--
 sysdeps/loongarch/nptl/bits/pthreadtypes-arch.h         | 6 +++---
 sysdeps/loongarch/nptl/bits/semaphore.h                 | 4 ++--
 sysdeps/loongarch/nptl/pthread-offsets.h                | 3 +--
 sysdeps/loongarch/nptl/pthreaddef.h                     | 4 ++--
 sysdeps/loongarch/nptl/tls.h                            | 4 ++--
 sysdeps/loongarch/setjmp.S                              | 4 ++--
 sysdeps/loongarch/sotruss-lib.c                         | 4 ++--
 sysdeps/loongarch/stackinfo.h                           | 6 +++---
 sysdeps/loongarch/tst-audit.h                           | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/atomic-machine.h      | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/clone.S               | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/dl-static.c           | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/init-first.c          | 3 +--
 sysdeps/unix/sysv/linux/loongarch/kernel-features.h     | 5 ++---
 sysdeps/unix/sysv/linux/loongarch/ldsodefs.h            | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/libc-vdso.h           | 3 +--
 sysdeps/unix/sysv/linux/loongarch/lp64/jmp_buf-macros.h | 3 +--
 sysdeps/unix/sysv/linux/loongarch/makecontext.c         | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/profil-counter.h      | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/sigcontextinfo.h      | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/sys/procfs.h          | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/sys/ucontext.h        | 4 ++--
 sysdeps/unix/sysv/linux/loongarch/syscall.c             | 4 ++--
 39 files changed, 76 insertions(+), 82 deletions(-)

diff --git a/sysdeps/loongarch/Makefile b/sysdeps/loongarch/Makefile
index b33673a9b2..d5801b3ce5 100644
--- a/sysdeps/loongarch/Makefile
+++ b/sysdeps/loongarch/Makefile
@@ -2,7 +2,7 @@ ifeq ($(subdir),misc)
 sysdep_headers += sys/asm.h
 endif
 
-# RISC-V's assembler also needs to know about PIC as it changes the definition
+# LoongArch's assembler also needs to know about PIC as it changes the definition
 # of some assembler macros.
 ASFLAGS-.os += $(pic-ccflag)
 CFLAGS-elf-init.oS += -mcmodel=large
diff --git a/sysdeps/loongarch/__longjmp.S b/sysdeps/loongarch/__longjmp.S
index 788421f724..f556157371 100644
--- a/sysdeps/loongarch/__longjmp.S
+++ b/sysdeps/loongarch/__longjmp.S
@@ -1,5 +1,5 @@
-/* longjmp, RISC-V version.
-   Copyright (C) 1996-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/bits/link.h b/sysdeps/loongarch/bits/link.h
index 149e6b87a2..554dfdc06d 100644
--- a/sysdeps/loongarch/bits/link.h
+++ b/sysdeps/loongarch/bits/link.h
@@ -1,5 +1,5 @@
-/* Machine-specific declarations for dynamic linker interface.  RISC-V version.
-   Copyright (C) 2005-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -28,7 +28,7 @@ typedef struct La_loongarch_regs
   unsigned long int lr_sp;
 } La_loongarch_regs;
 
-/* Return values for calls from PLT on RISC-V.  */
+/* Return values for calls from PLT on LoongArch.  */
 typedef struct La_loongarch_retval
 {
   unsigned long int lrv_a0;
diff --git a/sysdeps/loongarch/bits/setjmp.h b/sysdeps/loongarch/bits/setjmp.h
index 262fe07911..92711def51 100644
--- a/sysdeps/loongarch/bits/setjmp.h
+++ b/sysdeps/loongarch/bits/setjmp.h
@@ -1,5 +1,5 @@
-/* Define the machine-dependent type `jmp_buf'.  RISC-V version.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/dl-machine.h b/sysdeps/loongarch/dl-machine.h
index 2e9604f7c9..c686cd7169 100644
--- a/sysdeps/loongarch/dl-machine.h
+++ b/sysdeps/loongarch/dl-machine.h
@@ -1,5 +1,5 @@
-/* Machine-dependent ELF dynamic relocation inline functions.  RISC-V version.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -60,7 +60,7 @@
 static inline int __attribute_used__
 elf_machine_matches_host (const ElfW(Ehdr) *ehdr)
 {
-  /* We can only run RISC-V binaries.  */
+  /* We can only run LoongArch binaries.  */
   if (ehdr->e_machine != EM_LOONGARCH)
     return 0;
 
diff --git a/sysdeps/loongarch/dl-tls.h b/sysdeps/loongarch/dl-tls.h
index 3c8f481be0..70110c50bf 100644
--- a/sysdeps/loongarch/dl-tls.h
+++ b/sysdeps/loongarch/dl-tls.h
@@ -1,5 +1,5 @@
-/* Thread-local storage handling in the ELF dynamic linker.  RISC-V version.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/dl-trampoline.S b/sysdeps/loongarch/dl-trampoline.S
index 81e8e5be1d..1f4689e067 100644
--- a/sysdeps/loongarch/dl-trampoline.S
+++ b/sysdeps/loongarch/dl-trampoline.S
@@ -1,5 +1,4 @@
-/* RISC-V PLT trampoline
-   Copyright (C) 2017-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
 
    This file is part of the GNU C Library.
 
diff --git a/sysdeps/loongarch/gccframe.h b/sysdeps/loongarch/gccframe.h
index 5f500c500d..5c799c6474 100644
--- a/sysdeps/loongarch/gccframe.h
+++ b/sysdeps/loongarch/gccframe.h
@@ -1,5 +1,5 @@
-/* Definition of object in frame unwind info.  RISC-V version.
-   Copyright (C) 2001-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/jmpbuf-offsets.h b/sysdeps/loongarch/jmpbuf-offsets.h
index fff4d0d9c6..bc4c1523b6 100644
--- a/sysdeps/loongarch/jmpbuf-offsets.h
+++ b/sysdeps/loongarch/jmpbuf-offsets.h
@@ -1,5 +1,5 @@
-/* Private macros for accessing __jmp_buf contents.  RISC-V version.
-   Copyright (C) 2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/jmpbuf-unwind.h b/sysdeps/loongarch/jmpbuf-unwind.h
index 2e5f37f102..c866d9101d 100644
--- a/sysdeps/loongarch/jmpbuf-unwind.h
+++ b/sysdeps/loongarch/jmpbuf-unwind.h
@@ -1,5 +1,5 @@
-/* Examine __jmp_buf for unwinding frames.  RISC-V version.
-   Copyright (C) 2003-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/ldsodefs.h b/sysdeps/loongarch/ldsodefs.h
index 466434df6e..60b6db58f8 100644
--- a/sysdeps/loongarch/ldsodefs.h
+++ b/sysdeps/loongarch/ldsodefs.h
@@ -38,7 +38,7 @@ struct La_loongarch_retval;
 				       struct La_loongarch_retval *,	\
 				       const char *);
 
-/* The RISC-V ABI specifies that the dynamic section has to be read-only.  */
+/* The LoongArch ABI specifies that the dynamic section has to be read-only.  */
 
 #define DL_RO_DYN_SECTION 1
 
diff --git a/sysdeps/loongarch/libc-tls.c b/sysdeps/loongarch/libc-tls.c
index c72abc1d34..0b0590d189 100644
--- a/sysdeps/loongarch/libc-tls.c
+++ b/sysdeps/loongarch/libc-tls.c
@@ -1,5 +1,5 @@
-/* Thread-local storage handling in the ELF dynamic linker.  RISC-V version.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -19,7 +19,7 @@
 #include <csu/libc-tls.c>
 #include <dl-tls.h>
 
-/* On RISC-V, linker optimizations are not required, so __tls_get_addr
+/* On LoongArch, linker optimizations are not required, so __tls_get_addr
    can be called even in statically linked binaries.  In this case module
    must be always 1 and PT_TLS segment exist in the binary, otherwise it
    would not link.  */
diff --git a/sysdeps/loongarch/lp64/rvd/s_ceil.c b/sysdeps/loongarch/lp64/rvd/s_ceil.c
index af99ecca3d..b2e5943595 100644
--- a/sysdeps/loongarch/lp64/rvd/s_ceil.c
+++ b/sysdeps/loongarch/lp64/rvd/s_ceil.c
@@ -1,5 +1,5 @@
-/* ceil().  RISC-V version.
-   Copyright (C) 2017-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/lp64/rvd/s_floor.c b/sysdeps/loongarch/lp64/rvd/s_floor.c
index f34af98cce..8138239666 100644
--- a/sysdeps/loongarch/lp64/rvd/s_floor.c
+++ b/sysdeps/loongarch/lp64/rvd/s_floor.c
@@ -1,5 +1,5 @@
-/* floor().  RISC-V version.
-   Copyright (C) 2017-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/machine-gmon.h b/sysdeps/loongarch/machine-gmon.h
index a50d50a2e1..0b49082da9 100644
--- a/sysdeps/loongarch/machine-gmon.h
+++ b/sysdeps/loongarch/machine-gmon.h
@@ -1,5 +1,5 @@
-/* RISC-V definitions for profiling support.
-   Copyright (C) 1996-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/memusage.h b/sysdeps/loongarch/memusage.h
index 008295269a..bdf99f8a0f 100644
--- a/sysdeps/loongarch/memusage.h
+++ b/sysdeps/loongarch/memusage.h
@@ -1,5 +1,5 @@
-/* Machine-specific definitions for memory usage profiling, RISC-V version.
-   Copyright (C) 2000-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/nptl/bits/pthreadtypes-arch.h b/sysdeps/loongarch/nptl/bits/pthreadtypes-arch.h
index 760d8001c3..5a761355e2 100644
--- a/sysdeps/loongarch/nptl/bits/pthreadtypes-arch.h
+++ b/sysdeps/loongarch/nptl/bits/pthreadtypes-arch.h
@@ -1,5 +1,5 @@
-/* Machine-specific pthread type layouts.  RISC-V version.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -45,7 +45,7 @@
 #define __ONCE_ALIGNMENT
 
 /* There is a lot of padding in this structure.  While it's not strictly
-   necessary on RISC-V, we're going to leave it in to be on the safe side in
+   necessary on LoongArch, we're going to leave it in to be on the safe side in
    case it's needed in the future.  Most other architectures have the padding,
    so this gives us the same extensibility as everyone else has.  */
 struct __pthread_rwlock_arch_t
diff --git a/sysdeps/loongarch/nptl/bits/semaphore.h b/sysdeps/loongarch/nptl/bits/semaphore.h
index 2fed17d0e5..a9ddefb2af 100644
--- a/sysdeps/loongarch/nptl/bits/semaphore.h
+++ b/sysdeps/loongarch/nptl/bits/semaphore.h
@@ -1,5 +1,5 @@
-/* Machine-specific POSIX semaphore type layouts.  RISC-V version.
-   Copyright (C) 2002-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/nptl/pthread-offsets.h b/sysdeps/loongarch/nptl/pthread-offsets.h
index 5f7031834f..04130879cf 100644
--- a/sysdeps/loongarch/nptl/pthread-offsets.h
+++ b/sysdeps/loongarch/nptl/pthread-offsets.h
@@ -1,5 +1,4 @@
-/* RISC-V pthread offsets
-   Copyright (C) 2017-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
 
    This file is part of the GNU C Library.
 
diff --git a/sysdeps/loongarch/nptl/pthreaddef.h b/sysdeps/loongarch/nptl/pthreaddef.h
index 442422aefb..87c407bc89 100644
--- a/sysdeps/loongarch/nptl/pthreaddef.h
+++ b/sysdeps/loongarch/nptl/pthreaddef.h
@@ -1,5 +1,5 @@
-/* pthread machine parameter definitions,  RISC-V version.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/nptl/tls.h b/sysdeps/loongarch/nptl/tls.h
index 46250108d0..8d2d4ca223 100644
--- a/sysdeps/loongarch/nptl/tls.h
+++ b/sysdeps/loongarch/nptl/tls.h
@@ -1,5 +1,5 @@
-/* Definition for thread-local data handling.  NPTL/RISC-V version.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/setjmp.S b/sysdeps/loongarch/setjmp.S
index 21c0d9eb7e..d5a7dd4fa1 100644
--- a/sysdeps/loongarch/setjmp.S
+++ b/sysdeps/loongarch/setjmp.S
@@ -1,5 +1,5 @@
-/* setjmp for RISC-V.
-   Copyright (C) 1996-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/sotruss-lib.c b/sysdeps/loongarch/sotruss-lib.c
index 98357004b6..124db4406e 100644
--- a/sysdeps/loongarch/sotruss-lib.c
+++ b/sysdeps/loongarch/sotruss-lib.c
@@ -1,5 +1,5 @@
-/* Override generic sotruss-lib.c to define actual functions for RISC-V.
-   Copyright (C) 2012-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/loongarch/stackinfo.h b/sysdeps/loongarch/stackinfo.h
index 5b4f91c6a1..5f5e6ad1e6 100644
--- a/sysdeps/loongarch/stackinfo.h
+++ b/sysdeps/loongarch/stackinfo.h
@@ -1,5 +1,5 @@
-/* Stack environment definitions for RISC-V.
-   Copyright (C) 2011-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -24,7 +24,7 @@
 
 #include <elf.h>
 
-/* On RISC-V the stack grows down.  */
+/* On LoongArch the stack grows down.  */
 #define _STACK_GROWS_DOWN	1
 
 /* Default to a non-executable stack.  */
diff --git a/sysdeps/loongarch/tst-audit.h b/sysdeps/loongarch/tst-audit.h
index 7ff48f4cb4..d8d260eb93 100644
--- a/sysdeps/loongarch/tst-audit.h
+++ b/sysdeps/loongarch/tst-audit.h
@@ -1,5 +1,5 @@
-/* Definitions for testing PLT entry/exit auditing.  RISC-V version.
-   Copyright (C) 2005-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/atomic-machine.h b/sysdeps/unix/sysv/linux/loongarch/atomic-machine.h
index 3082b187f0..ac1948eade 100644
--- a/sysdeps/unix/sysv/linux/loongarch/atomic-machine.h
+++ b/sysdeps/unix/sysv/linux/loongarch/atomic-machine.h
@@ -1,5 +1,5 @@
-/* Low-level functions for atomic operations. RISC-V version.
-   Copyright (C) 2014-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/clone.S b/sysdeps/unix/sysv/linux/loongarch/clone.S
index 4e8976320b..8d689ead44 100644
--- a/sysdeps/unix/sysv/linux/loongarch/clone.S
+++ b/sysdeps/unix/sysv/linux/loongarch/clone.S
@@ -1,5 +1,5 @@
-/* Wrapper around clone system call.  RISC-V version.
-   Copyright (C) 1996-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/dl-static.c b/sysdeps/unix/sysv/linux/loongarch/dl-static.c
index b2fbb171a7..12b030f0f9 100644
--- a/sysdeps/unix/sysv/linux/loongarch/dl-static.c
+++ b/sysdeps/unix/sysv/linux/loongarch/dl-static.c
@@ -1,5 +1,5 @@
-/* Variable initialization.  RISC-V version
-   Copyright (C) 2001-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/init-first.c b/sysdeps/unix/sysv/linux/loongarch/init-first.c
index 4e8bbdbb36..5185a831d0 100644
--- a/sysdeps/unix/sysv/linux/loongarch/init-first.c
+++ b/sysdeps/unix/sysv/linux/loongarch/init-first.c
@@ -1,5 +1,4 @@
-/* RISC-V VDSO initialization
-   Copyright (C) 2017-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
 
    This file is part of the GNU C Library.
 
diff --git a/sysdeps/unix/sysv/linux/loongarch/kernel-features.h b/sysdeps/unix/sysv/linux/loongarch/kernel-features.h
index d6c1c90a1b..c87c796754 100644
--- a/sysdeps/unix/sysv/linux/loongarch/kernel-features.h
+++ b/sysdeps/unix/sysv/linux/loongarch/kernel-features.h
@@ -1,6 +1,5 @@
-/* Set flags signalling availability of kernel features based on given
-   kernel version number.  RISC-V version.
-   Copyright (C) 2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+ *
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/ldsodefs.h b/sysdeps/unix/sysv/linux/loongarch/ldsodefs.h
index b9421ffdfa..c0fc704647 100644
--- a/sysdeps/unix/sysv/linux/loongarch/ldsodefs.h
+++ b/sysdeps/unix/sysv/linux/loongarch/ldsodefs.h
@@ -1,5 +1,5 @@
-/* RISC-V dynamic linker data structures for loaded ELF shared objects.
-   Copyright (C) 2001-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/libc-vdso.h b/sysdeps/unix/sysv/linux/loongarch/libc-vdso.h
index 1d44b34e02..658c27a55a 100644
--- a/sysdeps/unix/sysv/linux/loongarch/libc-vdso.h
+++ b/sysdeps/unix/sysv/linux/loongarch/libc-vdso.h
@@ -1,5 +1,4 @@
-/* RISC-V VDSO function declarations
-   Copyright (C) 2017-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
 
    This file is part of the GNU C Library.
 
diff --git a/sysdeps/unix/sysv/linux/loongarch/lp64/jmp_buf-macros.h b/sysdeps/unix/sysv/linux/loongarch/lp64/jmp_buf-macros.h
index 5726ba041b..5cefbf4be0 100644
--- a/sysdeps/unix/sysv/linux/loongarch/lp64/jmp_buf-macros.h
+++ b/sysdeps/unix/sysv/linux/loongarch/lp64/jmp_buf-macros.h
@@ -1,5 +1,4 @@
-/* jump buffer constants for RISC-V
-   Copyright (C) 2017-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
 
    This file is part of the GNU C Library.
 
diff --git a/sysdeps/unix/sysv/linux/loongarch/makecontext.c b/sysdeps/unix/sysv/linux/loongarch/makecontext.c
index 33f0fe1188..ee6f9b5657 100644
--- a/sysdeps/unix/sysv/linux/loongarch/makecontext.c
+++ b/sysdeps/unix/sysv/linux/loongarch/makecontext.c
@@ -1,5 +1,5 @@
-/* Create new context.  RISC-V version.
-   Copyright (C) 2001-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/profil-counter.h b/sysdeps/unix/sysv/linux/loongarch/profil-counter.h
index 748019ee1b..6a3cc201b9 100644
--- a/sysdeps/unix/sysv/linux/loongarch/profil-counter.h
+++ b/sysdeps/unix/sysv/linux/loongarch/profil-counter.h
@@ -1,5 +1,5 @@
-/* Low-level statistical profiling support function.  Linux/RISC-V version.
-   Copyright (C) 1996-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/sigcontextinfo.h b/sysdeps/unix/sysv/linux/loongarch/sigcontextinfo.h
index 1cb339ffd6..2a86479567 100644
--- a/sysdeps/unix/sysv/linux/loongarch/sigcontextinfo.h
+++ b/sysdeps/unix/sysv/linux/loongarch/sigcontextinfo.h
@@ -1,5 +1,5 @@
-/* RISC-V definitions for signal handling calling conventions.
-   Copyright (C) 2000-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/sys/procfs.h b/sysdeps/unix/sysv/linux/loongarch/sys/procfs.h
index 51d7da6952..9ae06b408b 100644
--- a/sysdeps/unix/sysv/linux/loongarch/sys/procfs.h
+++ b/sysdeps/unix/sysv/linux/loongarch/sys/procfs.h
@@ -1,5 +1,5 @@
-/* Core image file related definitions, RISC-V version.
-   Copyright (C) 1996-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/sys/ucontext.h b/sysdeps/unix/sysv/linux/loongarch/sys/ucontext.h
index 44162a552f..a46e7897e2 100644
--- a/sysdeps/unix/sysv/linux/loongarch/sys/ucontext.h
+++ b/sysdeps/unix/sysv/linux/loongarch/sys/ucontext.h
@@ -1,5 +1,5 @@
-/* struct ucontext definition, RISC-V version.
-   Copyright (C) 1997-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
diff --git a/sysdeps/unix/sysv/linux/loongarch/syscall.c b/sysdeps/unix/sysv/linux/loongarch/syscall.c
index 3e1a5b8ba9..b06a528e8b 100644
--- a/sysdeps/unix/sysv/linux/loongarch/syscall.c
+++ b/sysdeps/unix/sysv/linux/loongarch/syscall.c
@@ -1,5 +1,5 @@
-/* system call interface.  Linux/RISC-V version.
-   Copyright (C) 2001-2018 Free Software Foundation, Inc.
+/* Copyright (C) 2020-2021 Free Software Foundation, Inc.
+
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
-- 
2.20.1

